Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.NetworkInformation;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
+            {
+                Console.WriteLine("What is your first name?");
+                var name = Console.ReadLine();
+
+                Console.WriteLine("What is your last name?");
+                var surname = Console.ReadLine();
+
+                Console.WriteLine("What is your age?");
+                var age = int.Parse(Console.ReadLine());
+
+                Console.WriteLine("What is your weight (in lbs)?"); //Murica
```

Comment:
What does Murica mean?

Reply:
It's the way stereotypical United States "patriots" say "America" Lol sorry, just some metric vs imperial units humor.